jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/UriCompliance.java

```diff
@@ -170,6 +170,14 @@ public String getDescription()
 
     private static final AtomicInteger __custom = new AtomicInteger();
     private static final List<UriCompliance> KNOWN_MODES = List.of(DEFAULT, LEGACY, RFC3986, UNAMBIGUOUS, UNSAFE);
+    private static final EnumSet<Violation> NO_VIOLATION = EnumSet.noneOf(Violation.class);
+
+    // This saves a AccessController.executePrivileged() call caused by
+    // EnumSet.noneOf() calling EnumSet.getUniverse().
+    public static EnumSet<Violation> emptyViolationsEnumSet()
```

Comment:
I would call this method `newViolationsEnumSet()` because the prefix `empty` typically refers to a constant, which this is not (for example `Optional.empty()`, `Collections.empty*()`, etc.).

Reply:
I've removed this method altogether.